0009-salt-working-groups.md

```diff
@@ -0,0 +1,121 @@
+
+If you want to lead or be part of a different working group then the ones mentioned above, Salt would be happy to include that as part of initial working groups, provided Salt receives enough interest from others to participate. 
+
+## Application Process
+
+Salt community members indicate their interest to be part of one or more working group(s) by filling out an application consisting of simple 4-5 question survey; Applicants indicate their preference for the type of working group(s) they want to join and the role they want to play: Captain or facilitator. If an application is received past the due date then the applicant will automatically be considered for the next term of the working groups. 
+
+## Election 
+
+Salt will conduct an election to choose the working group members in the following scenarios: 
+- When the number of applicants interested in joining the working groups exceeds the number that is ideal for a working group (3 to 9 members) 
+- 2 or more community members indicate their interest in being captain
+- If Salt doesn’t receive any eligible applicants for captain role, a member of the Salt open/core will be elected to be captain of that working group. 
+
+The rules for the election process are:  
+- Election for the first working group captain will be internal and voting will be done by the Salt core team. 
+- Elections for all subsequent working group captains and members will be voted on by the outgoing members of the current working group
+
+## Timelines 
+
+Based on the applications submitted, Salt chooses the final working group members 
+
+Below is the approximate timeline for how long each working group term will last. The actual timeline may differ. For example, the Testing and Release Working Group might decide to convene immediately after a major release and dissolve after the next major release. 
+
+![Working Group Timelines](./diagrams/Working-group-timelines.svg)
+
+
+
+*T0 is feature freeze date of current major release
+ T1 is feature freeze date of next major release
+
+
+Working Groups will meet once a month for 60 mins. Below are some guidelines for working group participants, although each of these groups may operate a little differently depending on their needs and workflow.
+
+## Working Group Structure  
+
+Each member of a working group will fill one of 3 roles: captain, participant or facilitator. 
+
+### Captain
+
+Each working group captain will help lead their respective working group. 
+
+#### Role of Captains
+- The captain helps shepherd a subset of features/critical fixes for a given release
+- The captain will help coordinate other contributors' work in their area
+- The captain will attend monthly working group meetings for their respective working group. SaltStack will set these up, but SaltStack will hold no restriction on any additional meetings the captain may deem necessary
+
+#### Captain Guidelines
+
+- The captain will have merge rights to Salt repository
```

Comment:
Are there security guidelines for this sort of thing? For example, is there a document which outlines that those with merge access should have 2FA enabled?

Reply:
@Ch3LL This might be something we should add to Merging Guidelines SEP ?

Reply:
added here: https://github.com/saltstack/salt-enhancement-proposals/pull/13/files#diff-1f5ce66a12d7194668a8225607482bd9R47